.github/workflows/ci-sglang-benchmark.yml

```diff
@@ -70,8 +70,8 @@ jobs:
           # We could also pin to a known working or stable version.
           # This should eventually stabilize. Do the best we can for now.
           pip install -f https://iree.dev/pip-release-links.html --upgrade \
-            iree-base-compiler==3.0.0rc20241118 \
-            iree-base-runtime==3.0.0rc20241118 \
+            iree-base-compiler==3.1.0rc20241203 \
+            iree-base-runtime==3.1.0rc20241203 \
```

Comment:
I may want to revert the changes to `ci-sglang-benchmark.yml` and `ci-sdxl.yml` (neither was tested on `pull_request` here), but it would be nice to make this version pin update uniformly across all workflows 🤔 . Thoughts, @stbaione ?

Reply:
Ideally this would be pinned uniformly with the other workflows. However, due to the [iree regression](https://github.com/iree-org/iree/issues/19360) in this specific candidate release, it would cause the benchmark test to fail from the shortfin server crashing. Saw crashes on #602 after updating to this version, which was tested on `pull_request` a few times.

It might make more sense to keep this pinned with a known good version, and add a `#TODO` to update once regressions are fixed. Although, it may also make sense to change the version to `3.1.0rc20241202` instead. There are previous CI runs for tests like [ci - sharktank perplexity](https://github.com/nod-ai/shark-ai/actions/runs/12134607637/job/33832116539#step:5:190), which pass for `20241202`, but fail for `20241203`. Above mentioned issue may disqualify `1203` as a "stable release"

Reply:
Looks like the commit responsible for failures in `20241203` was reverted in [cb5be1dbd3560f692578c137eadbb413b41e44c7](https://github.com/iree-org/iree/commit/cb5be1dbd3560f692578c137eadbb413b41e44c7), causing [ci - sharktank perplexity short](https://github.com/nod-ai/shark-ai/actions/runs/12163940676/job/33924371706?pr=602) and [Llama Benchmarking 8B Tests](https://github.com/nod-ai/shark-ai/actions/runs/12163940666/job/33924371796?pr=602) to pass again.

So we could instead upgrade to `20241204` as well

Reply:
Okay, will try that.